docs/cluster-agent/HORIZONTAL_POD_AUTOSCALING.md

```diff
@@ -0,0 +1,292 @@
+# Datadog Cluster Agent - DCA | Getting Started
+
+The DCA is a **beta** feature, if you are facing any issues please reach out to our [support team](http://docs.datadoghq.com/help)
```

Comment:
Maybe "The Datadog Cluster Agent (DCA)..." and remove DCA from the title, then you're free to use the acronym throughout the doc.